internal/test/util/contains.go

```diff
@@ -0,0 +1,34 @@
+package util
+
+import "reflect"
+
+func Contains(list interface{}, elem interface{}) bool {
+	v := reflect.ValueOf(list)
+	for i := 0; i < v.Len(); i++ {
+		if reflect.DeepEqual(v.Index(i).Interface(), elem) {
+			return true
+		}
+	}
+	return false
+}
+
+func ContainsAll(list interface{}, elems interface{}) bool {
+	listV := reflect.ValueOf(list)
+	listLen := listV.Len()
+	elemsV := reflect.ValueOf(elems)
+	elemsLen := elemsV.Len()
+	if listLen < elemsLen {
+		return false
+	}
```

Comment:
This means that `ContainsAll([1], [1,1]) == false`, which might cause confusion.
It's a shame golang doesn't allow you to create generic sets.

Reply:
I guess this is just a property of the function - it could be said that `[1]` does not contain all the elements of `[1, 1]` (depending on how you're looking at it)